packages/components/tests/lib/controllers/app-controller.test.js

```diff
@@ -1,839 +1,1389 @@
+    expect(appC._store._mutations['app/resetWordData']).toBeDefined()
+    expect(appC._store._mutations['app/lexicalRequestFinished']).toBeDefined()
+    expect(appC._store._mutations['app/setHomonym']).toBeDefined()
+    expect(appC._store._mutations['app/setWordUsageExampleEnabled']).toBeDefined()
+    expect(appC._store._mutations['app/setInflData']).toBeDefined()
+    expect(appC._store._mutations['app/resetInflData']).toBeDefined()
+    expect(appC._store._mutations['app/setUpdatedGrammar']).toBeDefined()
+    expect(appC._store._mutations['app/setWordUsageExamplesReady']).toBeDefined()
+    expect(appC._store._mutations['app/setWordUsageAuthorsReady']).toBeDefined()
+    expect(appC._store._mutations['app/setWordLists']).toBeDefined()
+    expect(appC._store._mutations['app/shortDefsUpdated']).toBeDefined()
+    expect(appC._store._mutations['app/fullDefsUpdated']).toBeDefined()
+    expect(appC._store._mutations['app/setMorphDataReady']).toBeDefined()
+    expect(appC._store._mutations['app/setTranslDataReady']).toBeDefined()
+    expect(appC._store._mutations['app/setQueryStillActive']).toBeDefined()
+    expect(appC._store._mutations['app/setMouseMoveOverrideUpdate']).toBeDefined()
+  })
 
-    expect(uiC.panel.panelData.messages).toEqual([ 'foomessage1' ])
+  it('AppController - textSelectorParams: should return correct parameters for the "longTap" trigger on mobile', async () => {
+    appC = AppController.jestCreate(uiState, { textQueryTriggerMobile: 'longTap' })
+    await appC.init()
+    appC._platform = { isMobile: true, isDektop: false }
+    const [event, eventParams] = appC.textSelectorParams
+    expect(event.name).toBe('LongTap')
+    expect(eventParams).toBeUndefined()
+  })
+
+  it('AppController - textSelectorParams: should return correct parameters for the "longtap" trigger on mobile', async () => {
+    appC = AppController.jestCreate(uiState, { textQueryTriggerMobile: 'longtap' })
+    await appC.init()
+    appC._platform = { isMobile: true, isDektop: false }
+    const [event, eventParams] = appC.textSelectorParams
+    expect(event.name).toBe('LongTap')
+    expect(eventParams).toBeUndefined()
+  })
 
-    uiC.addMessage('foomessage2')
-    expect(uiC.panel.panelData.messages).toEqual([ 'foomessage1', 'foomessage2' ])
+  it('AppController - textSelectorParams: should return correct parameters for the null trigger on mobile', async () => {
+    appC = AppController.jestCreate(uiState, { textQueryTriggerMobile: null })
+    await appC.init()
+    appC._platform = { isMobile: true, isDektop: false }
+    const [event, eventParams] = appC.textSelectorParams
+    expect(event.name).toBe('LongTap')
+    expect(eventParams).toBeUndefined()
+  })
 
-    uiC.addImportantMessage('fooImportant1')
-    expect(uiC.panel.panelData.messages).toEqual([ 'foomessage1', 'foomessage2', 'fooImportant1' ])
+  it('AppController - textSelectorParams: should return correct parameters no trigger is specified on mobile', async () => {
```

Comment:
this test looks identical to the previous -- I think the create should be taking instead an empty object for options?

Reply:
Oh, yes, I've overlooked that. Thanks for catching! Will fix.